### DISCUSSION TITLE
ChromecastPi

### DISCUSSION THREAD

**Discussion by rickballard** [micro-nova/AmpliPi#322] [Ideas]
New functionality for the raspberry Pi... [ChromePi](https://www.howtogeek.com/803133/plex-will-turn-your-raspberry-pi-into-a-diy-chromecast-audio/) 

Add this to a $50 stereo amp and you have a zone controller that integrates with Google Home 

Does thls impact the market positioning for AmpliPi, and is there any way to leverage this capability to enhance AmpliPi functionality (and sales)?

**Comment by linknum23** [comment]
Awesome, it looks like they are starting to prototype PlexAmp support
again. If we are lucky we will be able to include it in the next release.

**Comment by Lohrer** [comment]
Yes we've been eagerly awaiting headless PlexAmp again! Updates are being posted in issue #200.
